packages/base/Container/src/Container/Container.tsx

```diff
@@ -118,62 +97,39 @@ export const Container: ContainerProps = documentable(
         // Avoid passing external classes inside the rest props
         /* eslint-disable @typescript-eslint/no-unused-vars */
         classes: externalClasses,
-        /* eslint-enable */
         ...rest
       } = props
 
-      const classes = useStyles(props)
-      const { className: responsiveClasses, style: responsiveStyle } =
-        useResponsiveProps({
-          'margin-top': filterOutStringAndPicassoSpacing(top),
-          'margin-bottom': filterOutStringAndPicassoSpacing(bottom),
-          'margin-left': filterOutStringAndPicassoSpacing(left),
-          'margin-right': filterOutStringAndPicassoSpacing(right),
-          padding: filterOutStringAndPicassoSpacing(padded),
-          gap: filterOutStringAndPicassoSpacing(gap),
-        })
-
-      const baseSpacingClasses = getBaseSpacingClasses(
-        { top, left, bottom, right, gap, padded },
-        classes
-      )
+      const spacingProps = { gap, padded, top, bottom, right, left }
 
       return (
         <Component
           {...rest}
           ref={ref}
-          className={cx(
-            classes[`${variant}Variant`],
-            {
-              [classes[`${padded}Padding`]]: typeof padded === 'string',
-              [classes[`${gap}Gap`]]: typeof gap === 'string',
-
-              [classes[`top${top}Margin`]]: typeof top === 'string',
-              [classes[`bottom${bottom}Margin`]]: typeof bottom === 'string',
-              [classes[`left${left}Margin`]]: typeof left === 'string',
-              [classes[`right${right}Margin`]]: typeof right === 'string',
-
-              [classes[`${align}TextAlign`]]: typeof align === 'string',
-
-              [classes[`${kebabToCamelCase(alignItems || '')}AlignItems`]]:
-                alignItems,
-
-              [classes[
-                `${kebabToCamelCase(justifyContent || '')}JustifyContent`
-              ]]: justifyContent,
-
-              [classes.bordered]: bordered,
-              [classes.rounded]: rounded,
-              [classes.flex]: flex,
-              [classes.inline]: inline,
-              [classes[kebabToCamelCase(direction || '')]]:
-                direction && direction !== 'row',
-            },
-            baseSpacingClasses,
-            responsiveClasses,
+          className={twMerge(
+            variant && variantClassesByColor[variant],
+            getSpacingClasses(spacingProps),
+
+            typeof align === 'string' && alignmentClasses.textAlign[align],
+
+            alignItems && alignmentClasses.alignItems[alignItems],
+
+            justifyContent && alignmentClasses.justifyContent[justifyContent],
+
+            bordered && 'border-DEFAULT border-solid border-gray-200',
```

Comment:
```suggestion
            bordered && 'border border-solid border-gray-200',
```
I believe the default border with no value should already provide the same outcome. `border-DEFAULT` is not a standard Tailwind directive